.circleci/config.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     executor:
       name: small-docker
     environment:
-      LIBRARY_NAME: permissions
+      LIBRARY_NAME: celery
```

Comment:
What things does this change affect?

Reply:
just checked, used for the python release process here https://github.com/eventbrite/docker-dev/blob/master/ci_cd/scripts/python/release.py#L480

Reply:
*old devpi release